src/servers/ZoneServer2016/zoneserver.ts

```diff
@@ -6790,15 +6790,16 @@ export class ZoneServer2016 extends EventEmitter {
 
   igniteoptionPass(client: Client) {
     for (const a in this._explosives) {
+      const explosive = this._explosives[a];
       if (
         isPosInRadius(
-          1,
+          2.0,
           client.character.state.position,
-          this._explosives[a].state.position
+          explosive.state.position
         )
       ) {
-        this._explosives[a].ignite(this, client);
-        return;
+        explosive.ignite(this, client);
+        Scheduler.wait(750);
```

Comment:
```suggestion
        Scheduler.wait(750);
```
why do you wanna await that much ?

Reply:
I wanted to make it so the IED's light 1 by 1, testing it some more though and it didn't even work, just delayed the ignition process. Gonna look into it some more